admin/app/components/solidus_admin/products/index/component.rb

```diff
@@ -9,9 +9,10 @@ def image_column(product)
     image = product.gallery.images.first or return
 
     link_to(
-      image_tag(image.url(:small), class: 'h-10 w-10 max-w-min rounded border border-gray-100 '),
+      image_tag(image.url(:small), class: 'h-10 w-10 max-w-min rounded border border-gray-100', alt: ''),
```

Comment:
Why are we considering the image as decorative here?

Reply:
Because it doesn't add any additional information. The product name and image are rendered in another column.